### PR TITLE
More checks

### PR DESCRIPTION
**Pull request by bcvery1**
- Added:
   - go.mod/go.sum
   - README.md
   - .gitignore
   - Makefile
 - Can build with `make`; due to needing CGO, this must be done per OS
 - Checking every lookup available from Go's `net` package
 - Trying to get `/info` and output the response body

### CODE REVIEW

**Review by lacroixthomas — COMMENTED** [review]
Do we really need to use an external dependency `github.com/bcvery1/gotime` just to format a date ?

File: main.go [hunk 1]
```diff
@@ -1,15 +1,176 @@
...
+	if srvPort == 0 {
+		log("Port is required. This should be the port of the Flix server, not including ':'")
+		os.Exit(1)
+	}
+	log("Address to lookup: %s", srvAddr)
+	log("Port to use: %d", srvPort)
+	if useTLS {
+		log("Using TLS")
+	} else {
+		log("No using TLS")
```
Comment (samsamm777) [inline main.go]:
```suggestion
		log("Not using TLS")
```

**Review by samsamm777 — CHANGES_REQUESTED** [review]

**Comment by bcvery1** [comment]
@samsamm777 @lacroixthomas updates done

**Review by lacroixthomas — APPROVED** [review]

**Review by samsamm777 — APPROVED** [review]

**Comment by samsamm777** [comment]
Thanks for doing this Ben. looking good!

**Pull request merged**